index.html

```diff
@@ -20,8 +22,8 @@ <h1>Travoogle</h1>
   <div class = "row " id="searchDiv">
       <div class="col-md-4 box"> 
         <div class="card">  
-            <input type="text" id="search"class="form-control" placeholder="Travel to.....">
-            <button id="submit" class="btn btn-warning">Search</button>
```

Comment:
we still need this button with the id of "search" for my js file to work.

Reply:
i would recommend adding a class to the original button rather than deleting this button.